x-pack/winlogbeat/module/powershell/test/testdata/800.evtx.golden.json

```diff
@@ -0,0 +1,75 @@
+      "kind": "event",
+      "module": "powershell",
+      "provider": "PowerShell",
+      "sequence": 17
+    },
+    "host": {
+      "name": "vagrant-2019"
+    },
+    "log": {
+      "level": "information"
+    },
+    "message": "Pipeline execution details for command line:         Add-Type -AssemblyName System.IO.Compression.FileSystem\n. \n\nContext Information: \n\tDetailSequence=1\n\tDetailTotal=1\n\n\tSequenceNumber=17\n\n\tUserId=VAGRANT-2019\\vagrant\n\tHostName=ConsoleHost\n\tHostVersion=5.1.17763.1007\n\tHostId=ac3c99ce-7983-4996-807e-6a689eaba50b\n\tHostApplication=powershell -executionpolicy bypass \u0026 { if (Test-Path variable:global:ProgressPreference){set-variable -name variable:global:ProgressPreference -value 'SilentlyContinue'};. c:/Windows/Temp/packer-ps-env-vars-5e5637dd-15a9-73e0-889a-c01f541a8bc6.ps1; \u0026'c:/Windows/Temp/script-5e5637dd-5626-019d-027a-02e78baaacc9.ps1'; exit $LastExitCode }\n\tEngineVersion=5.1.17763.1007\n\tRunspaceId=6a447a2c-693e-4d41-948d-129b455b2569\n\tPipelineId=1\n\tScriptName=C:\\Windows\\system32\\WindowsPowerShell\\v1.0\\Modules\\Microsoft.PowerShell.Archive\\Microsoft.PowerShell.Archive.psm1\n\tCommandLine=        Add-Type -AssemblyName System.IO.Compression.FileSystem\n \n\nDetails: \nCommandInvocation(Add-Type): \"Add-Type\"\nParameterBinding(Add-Type): name=\"AssemblyName\"; value=\"System.IO.Compression.FileSystem\"",
+    "powershell": {
+      "engine": {
+        "version": "5.1.17763.1007"
+      },
+      "pipeline_id": "1",
+      "process": {
+        "executable_version": "5.1.17763.1007",
+        "is_remote": false
+      },
+      "runspace_id": "6a447a2c-693e-4d41-948d-129b455b2569"
+    },
+    "process": {
+      "args": [
+        "powershell",
+        "-executionpolicy",
+        "bypass",
+        "\u0026",
+        "{",
+        "if",
+        "(Test-Path",
+        "variable:global:ProgressPreference){set-variable",
+        "-name",
+        "variable:global:ProgressPreference",
+        "-value",
+        "'SilentlyContinue'};.",
+        "c:/Windows/Temp/packer-ps-env-vars-5e5637dd-15a9-73e0-889a-c01f541a8bc6.ps1;",
+        "\u0026'c:/Windows/Temp/script-5e5637dd-5626-019d-027a-02e78baaacc9.ps1';",
+        "exit",
+        "$LastExitCode",
+        "}"
+      ],
+      "args_count": 17,
+      "command_line": "powershell -executionpolicy bypass \u0026 { if (Test-Path variable:global:ProgressPreference){set-variable -name variable:global:ProgressPreference -value 'SilentlyContinue'};. c:/Windows/Temp/packer-ps-env-vars-5e5637dd-15a9-73e0-889a-c01f541a8bc6.ps1; \u0026'c:/Windows/Temp/script-5e5637dd-5626-019d-027a-02e78baaacc9.ps1'; exit $LastExitCode }",
+      "entity_id": "ac3c99ce-7983-4996-807e-6a689eaba50b"
+    },
+    "winlog": {
+      "api": "wineventlog",
+      "channel": "Windows PowerShell",
+      "computer_name": "vagrant-2019",
+      "event_data": {
+        "CommandLine": "        Add-Type -AssemblyName System.IO.Compression.FileSystem",
```

Comment:
This `CommandLine` here feels very specific to PS itself. To me would make sense to move it into `powershell` namespace rather than the `process.command_line` defined in ECS. WDYT?